vignettes/trap_effort.Rmd

```diff
@@ -10,6 +10,7 @@ vignette: >
 ```{r, include = FALSE}
```

Comment:
I didn't have the time to dig into this as deeply as I wanted but the markdown results make sense and I like the way you combined the methods